Desktop/analysis/analysis.cpp

```diff
@@ -297,6 +300,8 @@ void Analysis::initialized(AnalysisForm* form, bool isNewAnalysis)
 		_status = Empty;
 	
 	connect(_analysisForm,			&AnalysisForm::helpMDChanged,		this,			&Analysis::helpMDChanged					);
+	connect(this,					&Analysis::RSourceChanged,	_analysisForm,	&AnalysisForm::rSourceChanged				);
```

Comment:
RSourceChanged -> rSourceChanged ?

Wouldve been nicer if they were the same...